tests/components/application-connector/wait-for-gateway-tests-finish.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+NAMESPACE=test
+TIMEOUT_IN_SECONDS=120
+
+echo "Waiting for tests to finish"
+elapsedSeconds=0
+
+status=$(kubectl get pod/application-gateway-test -n $NAMESPACE -ojsonpath='{ .status.phase}')
+until [[ $status == Succeeded || $status == Failed ]]; do
+  printf '.'
```

Comment:
Do we need to print those statuses? There are no timestamps being printed as well so in my opinion, it does not bring much value.

There is also a `.` printed at the start of each line, do we need it?

Reply:
The final idea is to leave printing the dot '.' sign every file second to show test progress.  But I have left this intentionally to provide more insight in the logs while running this job on prow only for the initial period after merging the code to main. If we got any problems - this message can show us why the pipeline fail and what is the state of the pod in case of failure.